src/interfaces/highs_lp_solver.f90

```diff
@@ -77,7 +77,7 @@ function Highs_passLp ( h, n, m, nz, cc, cl, cu, rl, ru, as, ai, av) result ( s
       integer ( c_int ) :: s
     end function Highs_passLp
 
-    function Highs_setOptionValue ( h, o, v ) result( s ) bind ( c, name='Highs_setOptionValue' )
+    function Highs_setOptionValue ( h, o, v ) result( s ) bind ( c, name='Highs_setHighsOptionValue' )
```

Comment:
Perhaps the function name should also be changed to match? That will make it easier to maintain/document/etc. (Because it was broken, no one can have used this, so it should be okay to change.)